packages/mui-joy/src/Autocomplete/Autocomplete.tsx

```diff
@@ -762,10 +762,10 @@ interface AutocompleteComponent {
 }
 
 Autocomplete.propTypes /* remove-proptypes */ = {
-  // ----------------------------- Warning --------------------------------
-  // | These PropTypes are generated from the TypeScript type definitions |
-  // |     To update them edit TypeScript types and run "yarn proptypes"  |
-  // ----------------------------------------------------------------------
+  // ┌────────────────────────────── Warning ──────────────────────────────┐
```

Comment:
Is this change related to the change ?
If not, they are creating tons of noise and could probably be extracted.

Reply:
It used to say "yarn proptypes", which I changed to "pnpm proptypes". The centering of the text and a different frame are not strictly related to this, but since I had to modify the text, I figured I might as well introduce this tiny improvement. I can revert it if you think it doesn't belong here.

Reply:
I would love if Github allowed us to easily mark all the files of a folder as "Viewed" :grimacing: 

We can keep it, it was purely to ease the reviewing of the PR and make sure not to miss an important change

Reply:
> I would love if Github allowed us to easily mark all the files of a folder as "Viewed"

It does! But not the standard interface, only the github.dev one: https://github.dev/mui/material-ui/pull/36287

Reply:
I just left a comment above too. The pull request is anyways heavy, it would be great if we can revert this particular change.